src/NServiceBus.RavenDB.Tests/Persistence/DocumentIds/DocumentIdConventionTestBase.cs

```diff
@@ -51,7 +51,7 @@ protected void ApplyTestConventions(IDocumentStore store, ConventionType type)
             ApplyConventionsInternal(store, type, false);
 
             var sagaTypes = new[] { typeof(TestSagaData) };
-            var conventions = new DocumentIdConventions(store, sagaTypes, EndpointName);
+            var conventions = new DocumentIdConventions(store, sagaTypes, EndpointName, true, true);
```

Comment:
Stupid nitpick, but I really prefer named parameters when there are more than a couple and especially when they're the same type.